projects/cmake/CMakeLists.txt

```diff
@@ -274,12 +274,19 @@ endforeach (EXAMPLE ${ADVANCED_SOURCES})
 # Kit Examples
 SET(ARM_SOURCES
```

Comment:
Just a note -- we'll still need to update the C++ API reference!  Let's see if we can get this released today.